osu.Game/Overlays/MusicController.cs

```diff
@@ -52,14 +53,13 @@ public class MusicController : CompositeDrawable
         [NotNull]
         public DrawableTrack CurrentTrack { get; private set; } = new DrawableTrack(new TrackVirtual(1000));
 
-        [Resolved]
-        private AudioManager audioManager { get; set; }
-
         [Resolved]
         private RealmAccess realm { get; set; }
 
         private readonly DrawableAudioMixer drawableAudioMixer;
 
+        public IAudioMixer AudioMixer => drawableAudioMixer;
```

Comment:
nit: Would rather call this `TrackMixer` or just `Mixer` (same applies for the field, name feels too long). Also should have xmldoc pointing out its purpose (and that it's layered on top of `CurrentTrack`).

Reply:
Done. Renamed it to `TrackMixer`.